numba_rvsdg/core/datastructures/basic_block.py

```diff
@@ -152,9 +152,9 @@ class PythonBytecodeBlock(BasicBlock):
         The bytecode offset immediately after the last bytecode of the block.
     """
 
-    begin: int = None
+    begin: int = 0
 
-    end: int = None
+    end: int = 0
```

Comment:
This can potentially be dangerous. The default value must be an invalid value for offset (`-1` maybe?). Otherwise, it'll be conflicting as well as hard to detect.